client_consumer/pulp/client/consumer/cli.py

```diff
@@ -316,8 +316,10 @@ def _delete_cert(self):
         id_cert_dir = self.context.config['filesystem']['id_cert_dir']
         id_cert_name = self.context.config['filesystem']['id_cert_filename']
         cert_filename = os.path.join(id_cert_dir, id_cert_name)
-        if os.path.exists(cert_filename):
-            os.remove(cert_filename)
+        server_rsa = self.context.config['server']['rsa_pub']
+        for filename in {cert_filename, server_rsa}:
```

Comment:
actually, it's even worse: since this is in the consumer CLI, it needs to run in python 2.4!